README.md

```diff
@@ -31,84 +31,84 @@ userPassword: readonly
 EOF
 ```
 
-Afterwards reload kidmd by running `systemctl reload kopano-kidmd`.
+Afterwards reload idmd by running `systemctl reload libregraph-idmd`.
 
 The `config.ldif` is for service users only and the data in there is used for bind requests only, but never returned for search requests.
 
 #### Main configuration
 
-The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the kidmd LDAP endpoint.
+The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the idmd LDAP endpoint.
 
-Whenever any of the ldif files are changed, added or removed, make sure to reload kidmd with `systemctl reload kopano-kidmd`.
+Whenever any of the ldif files are changed, added or removed, make sure to reload idmd with `systemctl reload libregraph-idmd`.
 
-Kidmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/kopano-kidmd/examples` directory.
+idmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/libregraph-idmd/examples` directory.
```

Comment:
Even if this sentence was there before, i thing the section about the "Example configuration" should go away as it is only true in relation to distro packaging and this READM should describe the source code.